drop-boxes/register-hlatyping-dropbox/register-hlatyping.py

```diff
@@ -68,26 +68,17 @@ def process(transaction):
         
     search_service = transaction.getSearchService()
     sc = SearchCriteria()
-    sc.addMatchClause(SearchCriteria.MatchClause.createAttributeMatch(SearchCriteria.MatchClauseAttribute.CODE, parentCode))
+    pc = SearchCriteria()
+    pc.addMatchClause(SearchCriteria.MatchClause.createAttributeMatch(SearchCriteria.MatchClauseAttribute.PROJECT, project));
+    sc.addSubCriteria(SearchSubCriteria.createExperimentCriteria(pc))
```

Comment:
These are the openbis criteria, right?
So you have a main searchcriteria that will create a new experiment if the condition of the other criteria is not met, correct?

Reply:
We need the space name to check for/create new samples (without the search service, which might not work for samples that have just been registered). Instead of searching a specific sample, I search for any sample of the project - that's why I need the project criterium. The space will be the same for any sample.